### PR TITLE
Add compose:material:material-icons-extended (jvm targets) to mavenLocal

### PR DESCRIPTION
**Pull request by dima-avdeev-jb**
Add **compose:material:material-icons-extended** libraries to mavenLocal publication

### CODE REVIEW

**Review by pjBooms — APPROVED** [review]

**Comment by eymar** [comment]
Is it needed for our samples? 
Do we need them for all targets (not only jvm)?

**Review by eymar — APPROVED** [review]

File: compose/build.gradle.kts [hunk 1]
```diff
@@ -27,6 +27,12 @@ val mainComponents =
         ComposeComponent(":compose:material:material"),
         ComposeComponent(":compose:material3:material3"),
         ComposeComponent(":compose:material:material-icons-core"),
+        ComposeComponent(":compose:material:material-icons-extended", supportedPlatforms = ComposePlatforms.JVM_BASED),
```
[the remark this answers is not in the document]
Comment (dima-avdeev-jb) [reply compose/build.gradle.kts]: Yeah, we allready have script publishComposeJbExtendedIconsToMavenLocal

**Comment by dima-avdeev-jb** [comment]
Simplify with only one new line

**Comment by dima-avdeev-jb** [comment]
> Is it needed for our samples? Do we need them for all targets (not only jvm)?

Yes, in sample **codeviewer**

**Comment by dima-avdeev-jb** [comment]
Split one big script into two small scripts.
In most cases, we need only the first script.
To publish Web DOM components, resources library and extended icons - we need a second script.

**Review by pjBooms — APPROVED** [review]

**Comment by dima-avdeev-jb** [comment]
> Is it needed for our samples? Do we need them for all targets (not only jvm)?

For now, only JVM works

**Pull request merged**